[DOCS] Small wording change (#11715)
great-expectations/great_expectations: docs/docusaurus/docs/cloud/expectations/manage_expectations.md

3. Get the Expectation from the Expectation Suite.

>    Expectations can be found within the Expectation Suite using a variety of methods. In the example below, the Expectation is found by iterating through all of the Expectations within the Expectation Suite and then matching the first Expectation on its class name and column name:

Reviewer: I find this phrasing a little hard to parse - I think "first" is referring to the first match, but the phrasing makes it sound like it's the first in the suite (which may or may not be the first match). Here's a suggestion for less ambiguous phrasing

```suggestion
   Expectations can be found within the Expectation Suite using a variety of methods. In the example below, the Expectation is found by iterating through all of the Expectations within the Expectation Suite and then selecting the first Expectation that matches the specified class name and column name:
```
